src/Features/Core/Portable/xlf/FeaturesResources.es.xlf

```diff
@@ -602,7 +602,7 @@ Asegúrese de usar el especificador "tt" para los idiomas para los que es necesa
       </trans-unit>
       <trans-unit id="Modifying_body_of_member_will_prevent_the_debug_session_from_continuing_due_to_internal_error">
         <source>Modifying the body of '{0}' will prevent the debug session from continuing due to internal error: {1}</source>
-        <target state="needs-review-translation">Si se modifica el cuerpo de "{0}", se impedirá que continúe la sesión de depuración debido a un error interno: {1}</target>
+        <target state="translated">Si se modifica el cuerpo de "{0}", se impedirá que continúe la sesión de depuración debido a un error interno: {1}.</target>
```

Comment:
Not sure, but I think `{1}` *may* already end in a punctuation. So this will have extra punctuation.